src/locales/de/modifier-type.ts

```diff
@@ -209,6 +209,9 @@ export const modifierType: ModifierTypeTranslationEntries = {
     "LEFTOVERS": { name: "Überreste", description: "Heilt 1/16 der maximalen KP eines Pokémon pro Runde" },
     "SHELL_BELL": { name: "Muschelglocke", description: "Heilt den Anwender um 1/8 des von ihm zugefügten Schadens" },
 
+    "TOXIC_ORB": { name: "Heiß-Orb", description: "It's a bizarre orb that exudes toxins when touched and will badly poison the holder during battle" },
+    "FLAME_ORB": { name: "Toxik-Orb", description: "It's a bizarre orb that gives off heat when touched and will affect the holder with a burn during battle" },
```

Comment:
These are also swapped

Reply:
It would need to be:
```suggestion
    "TOXIC_ORB": { name: "Heiß-Orb", description: "It's a bizarre orb that exudes toxins when touched and will badly poison the holder during battle" },
    "FLAME_ORB": { name: "Toxik-Orb", description: "It's a bizarre orb that gives off heat when touched and will affect the holder with a burn during battle" },
```
    "TOXIC_ORB": { name: "Toxik-Orb", description: "Ein Item zum Tragen. Dieser bizarre Orb vergiftet seinen Träger im Kampf schwer." },
    "FLAME_ORB": { name: "Heiß-Orb", description: "Ein Item zum Tragen. Dieser bizarre Orb fügt seinem Träger im Kampf Verbrennungen zu." },

Reply:
Source https://www.pokewiki.de/Toxik-Orb and https://www.pokewiki.de/Hei%C3%9F-Orb

Reply:
> It would need to be:
> ```suggestion
>     "TOXIC_ORB": { name: "Toxik-Orb", description: "Ein Item zum Tragen. Dieser bizarre Orb vergiftet seinen Träger im Kampf schwer." },
>     "FLAME_ORB": { name: "Heiß-Orb", description: "Ein Item zum Tragen. Dieser bizarre Orb fügt seinem Träger im Kampf Verbrennungen zu." },
> ```
> 

Please make it EXACTLY like i wrote it (unless it is to Long to fit). This is directly from the game

Reply:
> > It would need to be:
> 
> Please make it EXACTLY like i wrote it (unless it is to Long to fit). This is directly from the game

It was decided to remove this part of the description because in this game an object can only be held, there is no concept of a bag.